frontend/src/js/controllers/changePwdCtrl.js

```diff
@@ -7,67 +7,82 @@
         .module('evalai')
         .controller('ChangePwdCtrl', ChangePwdCtrl);
 
-    ChangePwdCtrl.$inject = ['utilities', 'validationSvc', '$state', '$http', '$rootScope'];
+    ChangePwdCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
 
-    function ChangePwdCtrl(utilities, validationSvc, $state, $http, $rootScope) {
+    function ChangePwdCtrl(utilities, $state, $http, $rootScope) {
         var vm = this;
         var userKey = utilities.getData('userKey');
         vm.wrnMsg = {};
         vm.isValid = {};
+        vm.user = {};
 
+        vm.startLoader = function(msg) {
+            $rootScope.isLoader = true;
+            $rootScope.loaderTitle = msg;
+            vm.loginContainer.addClass('low-screen');
```

Comment:
`vm.loginContainer` is not defined anywhere in this controller. So it will throw an error on `addClass`. Have you tested it in your local? See `AuthCtrl` for the reference.